### PR TITLE
Update docs for MxModelReflection 8.0.3

### PR DESCRIPTION
**Pull request by emirmx**

### CODE REVIEW

**Review by serg-firsov — DISMISSED** [review]
Looks good to me

**Comment by emirmx** [comment]
We will do some more updates. @MariaShaposhnikova @MarkvanMents

**Comment by MarkvanMents** [comment]
I'll set it to draft for now - please let me know when it is ready for another review.

**Review by serg-firsov — APPROVED** [review]

**Comment by emirmx** [comment]
> I'll set it to draft for now - please let me know when it is ready for another review.

Hi Mark. We updated the PR. It can be reviewed :-)

**Comment by MarkvanMents** [comment]
@emirmx I've proofread this and reformatted it a bit.
Can you check that it looks OK, then I can publish.

**Review by MarkvanMents — APPROVED** [review]
Looks good. Have reformatted it a bit so please let me know if it looks OK to publish.

File: content/en/docs/appstore/use-content/platform-supported-content/modules/model-reflection.md [hunk 1]
```diff
@@ -67,8 +69,19 @@ All the patterns use the locale from the context. Based on the user's language,
 
 ## Troubleshooting
 
-* Getting the following error: On click action must be set to "Do nothing" when "Selection" is enabled"  
-*Solution*: Update the "Data Widgets" module to latest version then re-import Mx Model Reflection.
+### Set On Click Action to "Do Nothing"
```
Comment (emirmx) [inline content/en/docs/appstore/use-content/platform-supported-content/modules/model-reflection.md]: This sounds like an instruction, can we make something similar to `You get `Set On Click Action to "Do Nothing"`  error.
or equivalent.

**Comment by emirmx** [comment]
> Looks good. Have reformatted it a bit so please let me know if it looks OK to publish.

Thanks @MarkvanMents Looks way nicer :-) I added just one comment.

File: content/en/docs/appstore/use-content/platform-supported-content/modules/model-reflection.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/appstore/use-content/platform-supported-content/modules/model-reflection.md]: I've added "Error Message" in front. Does that make it clearer?
d93a3bb1ecee77b6a8a62266cc4a3035c0b60b69

**Comment by emirmx** [comment]
@MarkvanMents Good to go now!

**Pull request merged**